sip-core/src/main/java/de/ikor/sip/foundation/core/declarative/orchestration/connector/ConnectorOrchestrator.java

```diff
@@ -35,19 +34,19 @@ public static ConnectorOrchestrator forConnector(final ConnectorDefinition relat
   }
 
   private void defaultRequestTransformer(final RouteDefinition definition) {
-    definition.log(
-        LoggingLevel.WARN,
-        String.format(
-            "No request transformation definition has been assigned to connector '%s' in connector-class '%s'",
-            relatedConnector.get().getId(), relatedConnector.getClass().getName()));
+    log.warn(
+        "No request transformation definition has been assigned to connector '{}' in connector-class '{}'",
+        relatedConnector.get().getId(),
+        relatedConnector.get().getClass().getName());
+    definition.process(exchange -> {});
```

Comment:
Do we need this empty processor?

Reply:
As currently implemented yes, otherwise route building would fail. Previously it contained just a log, which was spamming the console. It can be replaced by any other processor, but one must exist.